src/app/components/search/SaveList.js

```diff
@@ -274,16 +223,14 @@ const SaveList = ({
   };
 
   const handleClick = () => {
-    // From the "All Items" page and unmatched media page, we can just create a new list
-    if (objectType === 'all-items' || objectType === 'unmatched-media' || objectType === 'published') {
+    // From these pages we can just create a new list
```

Comment:
Suggestion: Add a `FIXME` to declare this at a higher level where we can reuse.

Reply:
Done